core/js/setupchecks.js

```diff
@@ -423,6 +423,24 @@
 							type: OC.SetupChecks.MESSAGE_TYPE_INFO
 						})
 					}
+					if (!data.isImagickEnabled) {
+						messages.push({
+							msg: t(
+								'core',
+								'Imagick is not enabled although the theming app is. For Favicon generation to work correctly, you need to install the imagick PHP extension.'
+							),
+							type: OC.SetupChecks.MESSAGE_TYPE_INFO
+						})
+					}
+					if (!data.areWebauthnExtensionsEnabled) {
+						messages.push({
+							msg: t(
+								'core',
+								'The PHP extensions gmp or bcmath are not enabled. For Webauthn to work correctly, please enable these extensions.'
```

Comment:
```suggestion
								'The PHP modules "gmp" and/or "bcmath" are not enabled. If you use WebAuthn passwordless authentication, these modules are required.'
```
Alignment and to clarify that these are required only when WebAuthn is actually used.

Reply:
lgtm :)